src/dg-testVault/.obsidian/plugins/obsidian-digital-garden/data.json

```diff
@@ -0,0 +1,46 @@
+{
+  "githubRepo": "testing-garden",
+  "githubToken": "THIS IS A MAJOR CYBERSECURITY ANTIPATTERN KEYS WILL BE LEAKED",
+  "githubUserName": "julesvirallinen",
+  "gardenBaseUrl": "",
+  "prHistory": [],
+  "baseTheme": "dark",
+  "theme": "{\"name\": \"default\", \"modes\": [\"dark\"]}",
+  "faviconPath": "",
+  "noteSettingsIsInitialized": true,
+  "siteName": "Digital Garden",
+  "slugifyEnabled": true,
+  "noteIconKey": "dg-note-icon",
+  "defaultNoteIcon": "",
+  "showNoteIconOnTitle": false,
+  "showNoteIconInFileTree": false,
+  "showNoteIconOnInternalLink": false,
+  "showNoteIconOnBackLink": false,
+  "showCreatedTimestamp": false,
+  "createdTimestampKey": "dg-created",
+  "showUpdatedTimestamp": false,
+  "updatedTimestampKey": "dg-updated",
+  "timestampFormat": "MMM dd, yyyy h:mm a",
+  "styleSettingsCss": "",
+  "pathRewriteRules": "F Folder:",
+  "customFilters": [
+    {
+      "pattern": "❄️",
+      "flags": "g",
+      "replace": "🌞"
+    }
```

Comment:
settings go here! 🌞

Reply:
As you mentioned, the githubtoken field being part of this is not great. But I've enabled push protection, meaning github should block anyone from pushing a token to this repository. But that still means people forking this won't have the same protection. Perhaps a good quick fix til we can think of something better is to add this file to .gitignore. At least then you would have to purposefully remove it from ignored if you need to push any changes.

Reply:
yeah that might work. I've noticed that git has gotten hard about having a file in the repo and gitignore at the same time. I think they've changed it a little because of secrets. We can try that yeah, or just a quick generator. OR some backdoor into the app or something to add it via an override file so it doesn't need to be set there.